specs/disciplina/educator/api/student.yaml

```diff
@@ -184,46 +216,62 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/SignedSubmission'
+              $ref: '#/components/schemas/Submission'
```

Comment:
> because `Submissions` are returned from GET `/submissions` methods, and witnesses should not be returned there

Okay, I thought it was perfectly fine to include `witness` for this endpoint.